src/Logging/DebugViewFriendlyErrorFormatter.php

```diff
@@ -127,12 +128,36 @@ public function output($statusCode)
         $output = $renderer->renderHeader();
         $output .= $renderer->renderInfo("Website Error", $this->getTitle(), $this->getBody());
 
-        if (Email::config()->admin_email) {
-            $mailto = Email::obfuscate(Email::config()->admin_email);
-            $output .= $renderer->renderParagraph('Contact an administrator: ' . $mailto . '');
+        if (!is_null($contactInfo = $this->addContactAdministratorInfo())) {
+            $output .= $renderer->renderParagraph($contactInfo);
         }
 
         $output .= $renderer->renderFooter();
         return $output;
     }
+
+    /**
+     * Generate the line with admin contact info
+     *
+     * @return ?string
```

Comment:
I’m not sure this is valid phpdoc, maybe use string|null

Reply:
done